radiotools/helper.py

```diff
@@ -63,6 +65,93 @@ def datetime_to_UTC(dt):
     return calendar.timegm(dt.timetuple())
 
 
+def get_local_zenith(pos):
+    """
+    Assumes spherical earth. Returns direction of zenith for given position "pos".
+    "pos" needs to be given in coordinate system with the origin at sea level.
+    """
+    origin = np.array([0, 0, atm.r_e])
+    return (origin + pos) / np.linalg.norm(origin + pos)
+
+
+def get_local_altitude(pos):
+    """
+    Assumes spherical earth. Returns height above sea level for position "pos".
+    "pos" needs to be given in coordinate system with the origin at sea level.
+    """
+    pos_tot = np.array([0, 0, atm.r_e]) + pos
+    return np.linalg.norm(pos_tot - get_local_zenith(pos) * atm.r_e)
+
+
+def get_local_zenith_angle(psource, preciever):
+    """
+    Assumes spherical earth. Returns zenith angle under which a reciever (preciever) sees a source (psource).
+    "preciever" and "psource" have to be in the same coordinate system with the origin at sea level.
+    """
+    local_zenith = get_local_zenith(preciever)
+    line = psource - preciever
+    return get_angle(local_zenith, line)
+
+
+def get_intersection_between_circle_and_line(r, b, c):
+    """
+    solution from: https://cp-algorithms.com/geometry/circle-line-intersection.html
+    """
+    a = 1  # without loss of generality set a to 1
+    eps = 1.e-6
+    r0 = (a ** 2 + b ** 2)
+    x0 = -a * c / r0
+    y0 = -b * c / r0
+    if c ** 2 > r ** 2 * r0 + eps:
+        # no intersection
+        return 0, 0
+    elif np.abs(c ** 2 - r ** 2 * r0) < eps:
+        # one intersection
+        return x0, y0
+    else:
+        # two intersections
+        d = r ** 2 - c ** 2 / r0
+        mult = np.sqrt(d / r0)
+        return x0 + b * mult, y0 - a * mult, x0 - b * mult, y0 + a * mult
+
+
+def get_zenith_angle_at_earth(zenith, station_level):
```

Comment:
function documentation is missing. Also in general, the input/output parameters should be defined with

```
Parameters
------------
variablea: float
    describtion of the variable

Reply:
I agree this is missing. I will add this during the next week.